lms/urls.py

```diff
@@ -1030,3 +1030,7 @@
 urlpatterns += [
     path('api/mfe_config/v1', include(('lms.djangoapps.mfe_config_api.urls', 'lms.djangoapps.mfe_config_api'), namespace='mfe_config_api'))
 ]
+
+urlpatterns += [
+    path("robots.txt", TemplateView.as_view(template_name="robots.txt", content_type="text/plain")),
```

Comment:
it feels wrong to use a template view here without including the template in this repo.

Reply:
moved robots.txt into this repo, closing https://github.com/Medality-Health/medality/pull/1269